en/advanced/nn_module_utilities.md

```diff
@@ -0,0 +1,31 @@
+# Neural Network Module Utilities
+
+This page will explain the parts of the module that does not directly concern something with the neural network, but PX4 related implementations, so that you more easily can shape the module to your needs.
+
+To learn more about how PX4 works in general, it is recommended to start with [Getting started](../dev_setup/getting_started.md).
+
+## Autostart
+
+In the PX4-Autopilot source code there are startup scripts, in these I have added a line for the mc_nn_control module. in ROMFS/px4fmu_common/init.d/rc.mc_apps it checks wether the module is included by looking for the parameter MC_NN_EN. If this is set to 1, which is the default value, the module will be started when booting PX4. Similarly you could create other parameters in the src/modules/mc_nn_control/mc_nn_control_params.c file for other startup script checks.
+
+:::info
+Note that it's only the first time you build that the default param value will overwrite the current one. So if you change it in the param file it will not be changed when flashing to the controller again. Todo this you can change it in QGC or in the [terminal](../modules/modules_command.md#param).
+:::
+
+## Custom Flight Mode
+The module creates its own flight mode "Neural Control" which lets you choose it from the flight mode menu in QGC and bind it to a switch on you RC controller. This is done by using the [ROS 2 Interface Library](../ros2/px4_ros2_interface_lib.md) internally. This involves several steps:
```

Comment:
When you say ROS 2 interface library - do you actually run ROS? Or do you mean that this is handled by the mavlink standard modes protocol (i.e. the internal API that the interface library uses).